### PR TITLE
[Keras Applications] WideResNet(s)

### PR DESCRIPTION
**Pull request by swghosh**
WideResNet(s) <sup>[1]</sup> are residual network structures that are aimed at improving classification performance by decreasing the depth and increasing the width of residual networks. Many SOTA paper(s) make use of WRN implementations in some form or the other (eg. <sup>[2], [3], [4]</sup>). It'd be useful to have WRN implemented in `tf.keras.applications` itself.

In case, WideResNet-28-10 and similar architectures can be added to `tf.keras.applications` we can look into the following:
1. Pre-trained weights for resized ImageNet  <sup>[5]</sup> (32 x 32) on WRN-28-10
2. Pre-trained weights for resized ImageNet <sup>[5]</sup> (64 x 64) on WRN-36-5 (https://patrykchrabaszcz.github.io/Imagenet32/#validation-performance)
3. Pre-trained weights for CIFAR-10 / SVHN? (although, keras_applications is tightly coupled with ImageNet related implementations)
4. Bottleneck ResNet-50-2 architecture with larger image sizes (eg. 224x224 / 299x299)

<sup>[1]</sup> [Wide Residual Networks](https://arxiv.org/abs/1605.07146)
<sup>[2]</sup> [SGDR: Stochastic Gradient Descent with Warm Restarts](https://arxiv.org/abs/1608.03983)
<sup>[3]</sup> [AutoAugment: Learning Augmentation Strategies From Data](https://arxiv.org/abs/1805.09501)
<sup>[4]</sup> [RandAugment: Practical automated data augmentation with a reduced search space](https://arxiv.org/abs/1909.13719)
<sup>[5]</sup> [A Downsampled Variant of ImageNet as an Alternative to the CIFAR datasets](https://arxiv.org/abs/1707.08819)

/cc: @tanzhenyu, @fchollet

### CODE REVIEW

**Comment by swghosh** [comment]
Hey @fchollet, any updates?

**Comment by swghosh** [comment]
Hey @fchollet, any thoughts on this?

**Comment by swghosh** [comment]
Gentle ping.
/cc: @fchollet, any thoughts on this?

**Comment by tanzhenyu** [comment]
We currently don't have the bandwidth to support this model. Closing it.
